Sources/ClientRuntime/Telemetry/Logging/SDKLoggingSystem.swift

```diff
@@ -18,7 +18,7 @@ public actor SDKLoggingSystem {
         factories[label] = logHandlerFactory
     }
 
-    public func initialize(defaultLogLevel: SDKLogLevel = .error) async {
+    public func initialize(defaultLogLevel: SDKLogLevel = .error) {
```

Comment:
This function has no async operations within, and async is removed.  Same for the `init` below.